src/OrchardCore.Modules/OrchardCore.Taxonomies/Controllers/AdminController.cs

```diff
@@ -98,10 +98,13 @@ public async Task<IActionResult> CreatePost(string id, string taxonomyContentIte
 
             var contentItem = await _contentManager.NewAsync(id);
 
-            var model = await _contentItemDisplayManager.UpdateEditorAsync(contentItem, _updateModelAccessor.ModelUpdater, true);
+            dynamic model = await _contentItemDisplayManager.UpdateEditorAsync(contentItem, _updateModelAccessor.ModelUpdater, true);
```

Comment:
Why you didn't use `var` as it was before?

Reply:
because the properties are added to a dynamic shape (i.e. to the properties dict). refer code above this